.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "java.compile.nullAnalysis.mode": "disabled",
+    "java.configuration.updateBuildConfiguration": "interactive",
+    "java.debug.settings.onBuildFailureProceed": true
+}
```

Comment:
You should exclude the .vscode directory from the repository (include in the .gitignore file). this is local to your setup only.